### PR TITLE
enhance: Support Array DataType for bulk_insert

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #28272 
Support array DataType for bulk_insert with json, binlog files.

### CODE REVIEW

**Comment by xiaocai2333** [comment]
rerun ut

**Comment by xiaocai2333** [comment]
reurn ut

**Comment by xiaocai2333** [comment]
reurn ut

File: pkg/util/merr/errors.go [hunk 1]
```diff
@@ -142,6 +142,9 @@ var (
 	// Do NOT export this,
 	// never allow programmer using this, keep only for converting unknown error to milvusError
 	errUnexpected = newMilvusError("unexpected error", (1<<16)-1, false)
+
+	// import
+	ErrImportFailed = newMilvusError("deserializing import data failed", 2100, false)
```
Comment (czs007) [inline pkg/util/merr/errors.go:147]: remove 'deserializing' ?

**Pull request merged**